js/my.js

```diff
@@ -232,3 +237,30 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  cmd = cmd.replace(/^\//, '');
+
+  var args = cmd.match(/(\S+) ?(\S+)? ?(.*)/);
+
+  if (args[1] === 'query' && args[2]) {
```

Comment:
Whenever using `match`, we must first always check the return value, as it may return `null`, at which point we'll get an error. `if (args && ...)`